temporalcli/commandsmd/commands.md

```diff
@@ -1,487 +1,1185 @@
+### temporal operator search-attribute list: List Search Attributes
 
-`temporal operator search-attribute list` displays a list of all Search Attributes.
+Display a list of active Search Attributes that can be assigned or used with 
+Workflow Queries.
 
-### temporal operator search-attribute remove: Removes custom search attribute metadata only
+```
+temporal operator search-attribute list
+```
+
+### temporal operator search-attribute remove: Remove custom Search Attributes
+
+Remove custom Search Attributes from the options that can be assigned or
+used with Workflow Queries.
+
+```
+temporal operator search-attribute remove \
+    --name YourAttributeName
+```
 
-`temporal operator search-attribute remove` command removes custom Search Attribute metadata.
+Remove without confirmation:
+
+```
+temporal operator search-attribute remove \
+    --name YourAttributeName \
+    --yes false
+```
 
 #### Options
 
-* `--name` (string[]) - Search Attribute name. Required.
-* `--yes`, `-y` (bool) - Confirm prompt to perform deletion.
+* `--name` (string[]) -
+  Search Attribute name.
+  Required.
+* `--yes`, `-y` (bool) -
+  Override confirmation request before removal.
+
+### temporal schedule: Perform operations on Schedules
 
-### temporal schedule: Perform operations on Schedules.
+Create, use, and update Schedules that allow Workflow Executions to
+be created at specified times:
 
-Schedule commands allow the user to create, use, and update Schedules.
-Schedules allow starting Workflow Execution at regular times.
+```
+temporal schedule [commands] [options]
+```
+
+For example:
+
+```
+temporal schedule describe \
+    --schedule-id "YourScheduleId"
+```
 
 #### Options
 
 Includes options set for [client](#options-set-for-client).
 
-### temporal schedule backfill: Backfills a past time range of actions.
+### temporal schedule backfill: Backfill past actions
 
- The `temporal schedule backfill` command runs the Actions that would have been run in a given time
-interval, all at once.
 
- You can use backfill to fill in Workflow Runs from a time period when the Schedule was paused, from
-before the Schedule was created, from the future, or to re-process an interval that was processed.
+Batch-execute actions that would have run during a specified time interval.
+Use this command to fill in Workflow Runs from when a Schedule was paused,
+from before a Schedule was created, from the future, or to re-process an
+interval that was previously executed.
 
-Schedule backfills require a Schedule ID, along with the time in which to run the Schedule. You can
-optionally override the overlap policy. It usually only makes sense to run backfills with either
-`BufferAll` or `AllowAll` (other policies will only let one or two runs actually happen).
+Backfills require a Schedule ID and the time period covered by the request. 
+It's best to use the `BufferAll` or `AllowAll` policies.
```

Comment:
We should briefly explain why. When reading documentation, I'm generally skeptical of "do this" / "don't do this" statement that aren't backed up by some kind of rationale, no matter how brief.

Reply:
1. Agreed.
2. I don't have a rationale. Researched, tried to find some, but the closest I've gotten is via Kapa AI, whose answer is pretty wordy. I need some specific wording and example help.

Here's my best start:
```
It's best to use the `BufferAll` or `AllowAll` policies as they are the 
most suitable for backfilling without conflict and without skipping and
therefore losing Workflow Executions.
```